.phpcs.xml.dist

```diff
@@ -79,4 +79,9 @@
 		</properties>
 	</rule>
 
+	<!-- Allow short @var assignments in tests. -->
+ 	<rule ref="Generic.Commenting.DocComment.MissingShort">
+    	<exclude-pattern>/tests/</exclude-pattern>
+ 	</rule>
```

Comment:
1. What is the benefit of adding this rule instead of going what phpcs and wpcs provides by default?
2. Why we are excluding `/tests/` and not following the consistencny?

Reply:
As stated by its comment, with this rule we allow short `@var` assignments  (which isn't allowed by default) only in tests. This allows us to use single-line `/** @var string */` docblocks instead of the following which takes multiple lines and is often not that useful:

```
/**
 * Variable.
 *
 * @var string
 */
```

We can of course still use it this form if we want to provide meaningful documentation in the first line. This was done so tests are not full of comment blocks such as the above. We still require the longer form in all other code.

Reply:
This short form syntax is good maybe we should use it in whole codebase (ofcourse not in this PR)